packages/react-components/react-accordion/stories/Accordion/AccordionSizes.stories.tsx

```diff
@@ -41,7 +41,7 @@ export const Sizes = () => (
 Sizes.parameters = {
   docs: {
     description: {
-      story: 'An accordion supports `small`, `medium`, `large` and `extra-large` sizes.',
+      story: 'An accordion supports `small`, `medium`, `large` and `extra-large` header sizes.',
```

Comment:
Should _Accordion_ and _Header_ be capitalized?
<code>Accordion supports `small`, `medium`, `large` and `extra-large` Header sizes
</code>

Reply:
Changed to "AccordionHeader supports..."